.travis.yml

```
@@ -292,13 +189,13 @@ jobs:
         - sudo ln -s /etc/nginx/sites-available/submitty.conf /etc/nginx/sites-enabled/submitty.conf
         - sudo service nginx restart
       script:
-        - echo "Authentication Method => $(sudo jq -r ".authentication_method" /usr/local/submitty/config/database.json)"
-        - TEST_URL="http://localhost" python3 -m unittest discover -v --start-directory tests
-        - sudo sed -ie "s/Pam/Database/g" /usr/local/submitty/config/database.json
-        - echo "Authentication Method => $(sudo jq -r ".authentication_method" /usr/local/submitty/config/database.json)"
-        - pushd tests
-        - TEST_URL="http://localhost" python3 -m unittest e2e.test_login
-        - popd
+       # - echo "Authentication Method => $(sudo jq -r ".authentication_method" /usr/local/submitty/config/database.json)"
+       # - TEST_URL="http://localhost" python3 -m unittest discover -v --start-directory tests
```

If having Travis do all this work anyway, might as well have it still test PAM authentication on the login tests:

```suggestion
       - echo "Authentication Method => $(sudo jq -r ".authentication_method" /usr/local/submitty/config/database.json)"
       - pushd tests
       - TEST_URL="http://localhost" python3 -m unittest e2e.test_login
       - popd
```